modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/download/task/DownloadEpisodeTask.kt

```diff
@@ -125,13 +126,13 @@ class DownloadEpisodeTask @AssistedInject constructor(
             this.episode = runBlocking { episodeManager.findEpisodeByUuid(episodeUUID!!) } ?: return Result.failure()
 
             if (this.episode.downloadTaskId != id.toString()) {
-                LogBuffer.e(LogBuffer.TAG_BACKGROUND_TASKS, "Mismatched download task id for episode ${episode.title}. Cancelling.")
+                LogBuffer.i(LogBuffer.TAG_BACKGROUND_TASKS, "Mismatched download task id for episode ${episode.title}. Cancelling.")
```

Comment:
There are a few issues reported in this class that are being sent to Sentry. If we aren't going address them I believe they should just be the log level of `info`.

Reply:
I agree with the other places you've made this change, but I wonder if we should keep this one since if it shot up all of a sudden that might be an issue we'd want to know about. I don't feel strongly though.

Aside from that, do you think it would be worth including the `downloadTaskId` and `id` in the error message. Seems like that would be helpful in tracking down why these are occurring. Just a thought, I don't feel strongly about this either.

Reply:
I have kept this and added the extra ids in the message. Thanks for the suggestions.